app/jobs/rerun_entry_job.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+##
+# This class is responsible for running a single bulkrax entry.
+#
+# @note This class is starting it's life in the Adventist repository but is being designed so that
+#       we could copy it rather easily into Bulkrax.
+#
+# separating this into a job will allow us to run multiple jobs concurrently in the background
+class RerunEntryJob < ApplicationJob
+  queue_as :reimport
+
+  ##
+  # @param entry [Entry Object] the entry to run
+  def perform(entry_class_name:, entry_id:)
+    bulkrax_entry = entry_class_name.constantize.find(entry_id)
+    Rails.logger.info("Submitting re-import for #{bulkrax_entry.class} ID=#{bulkrax_entry.id}")
+
+    begin
+      bulkrax_entry.build
+      bulkrax_entry.save
+    rescue StandardError => e
+      logger.error("Error re-submitting entry for #{bulkrax_entry.class} ID=#{bulkrax_entry.id}: #{e.message}")
```

Comment:
not sure if it needs it but maybe this?

```suggestion
      Rails.logger.error("Error re-submitting entry for #{bulkrax_entry.class} ID=#{bulkrax_entry.id}: #{e.message}")
```